soda_machine.rb

```diff
@@ -7,12 +7,24 @@ def initialize(args = {})
   end
 
   def current_inventory_count
+    @sodas.length
   end
 
   def find_soda(soda_brand)
+    @sodas.each do |a|
+        if a.brand == soda_brand
+          return a
+        end
+    end
+    nil
   end
```

Comment:
this is fine but check this out: 

```ruby
def find_soda(soda_brand)
  @sodas.each do |soda|
      return soda if soda.brand == soda_brand
  end

  nil
end
```

such simple edits and now it's much more readable and follows conventions of idiomatic ruby